package.json

```diff
@@ -26,13 +26,22 @@
   "devDependencies": {
     "@lifeomic/eslint-config-standards": "^2.1.1",
     "@lifeomic/jest-config": "^1.1.2",
+    "@lifeomic/logging": "^4.0.0",
     "@lifeomic/typescript-config": "^1.0.3",
     "@types/jest": "^27.4.1",
+    "@types/uuid": "^8.3.4",
     "eslint": "^8.9.0",
     "eslint-config-prettier": "^8.4.0",
     "jest": "^27.5.1",
     "prettier": "^2.5.1",
     "ts-jest": "^27.1.3",
-    "typescript": "^4.5.5"
+    "typescript": "^4.5.5",
+    "uuid": "^8.3.2"
+  },
+  "dependencies": {
+    "@types/aws-lambda": "^8.10.92"
```

Comment:
Shouldn't this be in dev deps?

Reply:
Good question -- based on the current code, it actually needs to be in `dependencies`, since one of the exports is typed using these types.

I _could_ put it in peerDeps, but didn't feel essential, and then it's "one less thing" for consumers to install.

In particular, this code requires it to be in `dependencies`: https://github.com/lifeomic/delta/pull/26/files#diff-984215d6bc9dea15d995039a1e743557ed07c22ceaa66a5b98576b68b07daf13R158